src/pages/academy/_recent-featured-posts.tsx

```diff
@@ -31,7 +31,7 @@ import { StandardImgWrapper } from './common/_styles'
 import { RecentDataType, FeaturedDataType } from './index'
 import { convertDate, getMinRead } from 'common/utility'
 import { QueryImage, Tabs, Header } from 'components/elements'
-import { localize, WithIntl } from 'components/localization'
+import { localize } from 'components/localization'
```

Comment:
You can remove `localize` function as well since we only support English in Academy.

Reply:
Thank u @mohammad-rework for your revision, I think what you said is out of the scope of this card, I just want to fix ts errors here.
do we need to check it here?

Reply:
If we are only focusing on TS errors we can skip it.